### PR TITLE
Dockerfile for xlearn in Python3.6

### PR DESCRIPTION
**Pull request by yueguoguo**
Dockerfile for building Docker image of `xlearn` with Python 3.6

### CODE REVIEW

**Comment by aksnzhy** [comment]
@yueguoguo Thanks for this contribution! I will review this PR as soon as possible!

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ENV USER=root
+
+# Upgrade or install packages needed
+RUN apt-get update && apt-get -y install \
+    g++ \
+    gcc \
+    binutils \
+    clang \ 
+    cmake \
+    protobuf-compiler
```
Comment (aksnzhy) [inline docker/Dockerfile]: Why we need protobuf here?

Comment (yueguoguo) [reply docker/Dockerfile]: Ah yea this can be removed.

**Comment by aksnzhy** [comment]
@yueguoguo Thanks for this contribution. I will add your icon to our page !

**Pull request merged**